Translations update from Weblate

weblate-siphalor opened the pull request:
Translations update from [Weblate](https://weblate.siphalor.de) for [Mouse Wheelie/Mouse Wheelie](https://weblate.siphalor.de/projects/mouse-wheelie/lang/).



Current translation status:

![Weblate translation status](https://weblate.siphalor.de/widgets/mouse-wheelie/-/lang/horizontal-auto.svg)

Siphalor commented on src/main/resources/assets/mousewheelie/lang/pt.json:
```diff
@@ -0,0 +1,78 @@
+{
```
Portugese is supposed to be Portugese (Brazil) - `pt_br`.
Gotta fix that.

Siphalor committed:
Add an action for fixing language file names

weblate committed:
Translated using Weblate (English)

Currently translated at 100.0% (97 of 97 strings)

Co-authored-by: Weblate <noreply@weblate.org>
Co-authored-by: Weblate Admin <info@siphalor.de>
Translate-URL: https://weblate.siphalor.de/projects/mouse-wheelie/lang/en/
Translation: Mouse Wheelie/Mouse Wheelie

weblate committed:
Translated using Weblate (German (de_DE))

Currently translated at 100.0% (97 of 97 strings)

Co-authored-by: Weblate <noreply@weblate.org>
Co-authored-by: theflusspferd <theflusspferd@gmail.com>
Translate-URL: https://weblate.siphalor.de/projects/mouse-wheelie/lang/de_DE/
Translation: Mouse Wheelie/Mouse Wheelie

weblate committed:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (97 of 97 strings)

Translated using Weblate (Portuguese)

Currently translated at 77.3% (75 of 97 strings)

Translated using Weblate (Portuguese)

Currently translated at 76.2% (74 of 97 strings)

Added translation using Weblate (Portuguese)

Co-authored-by: RiginDev <rigin.playhd@gmail.com>
Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.siphalor.de/projects/mouse-wheelie/lang/pt/
Translation: Mouse Wheelie/Mouse Wheelie

Siphalor committed:
Merge branch '1.16' into weblate-mouse-wheelie-lang
